v2/config/manager/manager_metrics_service.yaml

```diff
@@ -10,6 +10,6 @@ metadata:
 spec:
   ports:
     - name: metrics
-      port: 8080
+      port: 8443
```

Comment:
This is technically a breaking change but I think a good one.

We should remember to notify customers about it. Created #3846 to track this.